langserver/build_context.go

```diff
@@ -94,7 +107,13 @@ func ContainingPackage(bctx *build.Context, filename string) (*build.Package, er
 	if PathHasPrefix(filename, bctx.GOROOT) {
 		srcDir = bctx.GOROOT // if workspace is Go stdlib
 	} else {
-		srcDir = bctx.GOPATH
+		srcDir = "" // with no GOPATH, only stdlib will work
```

Comment:
Should the default value (if pkgDir is not a prefix of any gopath workspaces) be `bctx.GOROOT` or empty string?

On line 95/108 above, it says:

```Go
srcDir = bctx.GOROOT // if workspace is Go stdlib
```

So the empty string here seems unexpected, and I just wanted to confirm this is intentional.

I also see that the next line does `srcDir = path.Join(srcDir, "src") + "/"`, which I think would produce unexpected results if `srcDir` is empty string.